matplotcheck/tests/test_notebook.py

```diff
@@ -0,0 +1,145 @@
+"""Tests for the notebook module"""
+
+import pytest
+import matplotlib.pyplot as plt
+from matplotcheck.base import PlotTester
+import matplotcheck.notebook as nb
+
+
+@pytest.fixture
+def locals_dictionary_good():
+    """Abriged locals dictionary of a jupyter notebook with import in the proper
```

Comment:
@nkorinek i am not sure what this docstring means. what is this testing? what is a locals_dictionary?

Reply:
please add a sentence or two here about using `locals()` in a notebook! it will clarify everything! 

Abridged output of the `locals()` function which returns....